finality-aleph/src/sync/forest/mod.rs

```diff
@@ -133,6 +134,11 @@ impl<I: PeerId, J: Justification> VertexWithChildren<I, J> {
 // How deep can the forest be, vaguely based on two sessions ahead, which is the most we expect to
 // ever need worst case scenario.
 const MAX_DEPTH: u32 = 1800;
+#[allow(clippy::assertions_on_constants)]
```

Comment:
you can use https://docs.rs/static_assertions/latest/static_assertions/macro.const_assert.html here

Reply:
I meant it for the line below, not sure if this will help with the clippy :P

Reply:
we don't use this crate in `finality-aleph`, do we want to add it just for this?

Reply:
up to you, imho I think its nice addition and we could start using it in similar cases to this one. At the same time your approach is also fine

Reply:
ok, then let's wait for the second case :P